l1-contracts/src/core/libraries/Constants.sol

```diff
@@ -20,4 +20,19 @@ library Constants {
   uint256 internal constant CONTRACTS_PER_TX = 1;
   uint256 internal constant L2_TO_L1_MSGS_PER_TX = 2;
   uint256 internal constant L1_TO_L2_MSGS_PER_ROLLUP = 16;
+  uint256 internal constant KERNELS_PER_ROLLUP = 2;
```

Comment:
this could also be generated from the cpp script right? lets make a ticket and add it

Reply:
Are you talking about BASE_ROLLUPS as ROLLUPS? It gets pretty confusing if you remove that, as now it looks like you are trying to load all bytes in the rollup into a base later on.

Reply:
@rahul-kothari I don't think you handled this. The `ROLLUP` instead of `BASE_ROLLUP` or `BASE` make me think it is actually a root rollup which it is not.